ui/src/main/java/org/pentaho/mantle/client/dialogs/scheduling/ScheduleRecurrenceDialog.java

```diff
@@ -971,8 +971,14 @@ public void onResponseReceived( Request request, Response response ) {
                 callback.okPressed();
               }
             } else {
-              MessageDialogBox dialogBox = new MessageDialogBox( Messages.getString( "error" ),
-                Messages.getString( "serverErrorColon" ) + " " + response.getStatusCode(), false, false, true );
+              String message = response.getText();
```

Comment:
Ensure any messages from the server get displayed instead of just the server error code.